RULES.md

```diff
@@ -24,6 +24,7 @@ Please follow these simple rules:
 - The advertisement must be solely be based around a SA-MP Server
 - There must be NO unnecessary chat in the channel
 - Do not spam advertisements - only post once a week or risk being banned from advertising
+- Do not PM users from the server with advertisements. The #advertisements channel exists for a reason.
```

Comment:
I feel like that's kind of defensive sounding... Perhaps simply something like `Do not advertise outside of the #advertisements channel.`?

Reply:
It was in response to recent harassment.